### ISSUE TITLE
Outdated url pattern inside emote set's link

### ISSUE THREAD

**Issue by holysnipz** [melalawi/GlobalTwitchEmotes#32]
The base url of link to view emote sets under "Cached Emote Sets" still using old, outdated one.

--------------------

Twitchemotes.com, `/search/` is now using to find emotes, instead of channels.
![image](https://user-images.githubusercontent.com/9103583/84589425-9e098300-ae58-11ea-8bfe-2efcba674ba7.png)
 - *suggestion: either use `/channels/<channel's id>` which is the destination of `/search/channel` or just `/search/channel?query=<channel's name>` is fine.*

--------------------

BTTV, still using V2 link.
![image](https://user-images.githubusercontent.com/9103583/84589412-8a5e1c80-ae58-11ea-9280-1551cba7a853.png)

 - now you can see bttv's global emotes via this link: https://betterttv.com/emotes/global
 - *suggestion: maybe grab bttv's userid via the API then use it with `betterttv.com/users/<bttv's userid>` for better viewing experience.*

**Comment by melalawi** [comment]
Yeah I noticed this back when I updated the extension to support some of the newer emote APIs. It's in my backlog to fix.
